### ISSUE TITLE
Blog topic: state-of-the-art of natural language processing: case of 10-ks

### ISSUE THREAD

**Issue by jeremiahpslewis** [trr266/rescorptrans#8]
Hi all,

Is someone else interested in looking at this suggested topic? Think that at least three key aspects could be tackled here:

1. What algorithms do ppl currently use to attack this problem, are they generic to broader NLP or more specialized?

2. What does the rate of change look like? Where is the field likely to be in 5 years?

3. Will this still be a relevant problem in 5 years, or will open and structured data releases cannibalize need for NLP-based approaches?

**Comment by nrusha95** [comment]
Hi! I am also interested in working on a blog revolving around this topic. I indeed find the questions you mention quite interesting. I am interested in research on disclosures using textual analysis. Do you have any ideas about which data you would like to use to tackle these questions?

**Comment by jeremiahpslewis** [comment]
Tbh, no idea! ;) But we could start with a lit review and see what the status quo looks like?

**Comment by nrusha95** [comment]
Yes right :) I agree we have to decide on what the literature suggests

**Comment by Rajabalizadeh** [comment]
Hello. My name is Javad. I took the Research on Corporate Transparency course, too. My Ph.D. thesis topic is related to quantifying narrative disclosures (10-ks notes, MD&A sections, etc) with textual analysis. Before this, I have published some papers in this field (comment letters and readability, readability and informational efficiency, firm fundamentals and readability), and I am familiar with the related literature. Therefore, I am interested in this topic so.

**Comment by Rajabalizadeh** [comment]
Hello. I prepared some notes about NLP in the accounting and finance field. After reading them, please put your comments to work more on the topic.
In accounting and finance, the below contexts provide ample fodder for applying textual analysis technology:
- Public filings
- Earnings calls transcripts
- Online availability of news articles,
- MD&A of 10-K and 10-Q has the most meaningful data,
- Securities and Exchange Commission (SEC) filings,
- Text from social media

Also, in the above contexts, previous studies have used different measures depending on the research goal. Some of them are:
1. Word count
2. Word cloud
3. Word tree
4. Word search
5. Readability
6. Tone/sentiment analysis
7. Repetition
8. Self-Inclusive Language (SIL) 

Please see the below paper, too. It is very useful (online version).
"Revealing Research Themes and Trends in 30 Top‐ranking Accounting Journals: A Text‐mining Approach"
(https://onlinelibrary.wiley.com/doi/abs/10.1111/abac.12214)

**Comment by towitter** [comment]
Hi guys! From our survey, we have the following participants that show an interest in this topic (github handle in brackets, if provided):

Jeremiah (@jlewis91), Javad (@Rajabalizadeh), Jonas (@Jonas-Materna), Lazaros, Rusha (@nrusha95), Harry (@harrynnh), Mahmoud (@mdelshadi)

**Comment by harrynnh** [comment]
Hi all,
I came across these 2 articles that can be useful for the blog post.
- El‐Haj, M., Rayson, P., Walker, M., Young, S., & Simaki, V. (2019). In search of meaning: Lessons, resources and next steps for computational analysis of financial discourse. Journal of Business Finance & Accounting, 46(3-4), 265-306.
- Loughran, T., & McDonald, B. (2016). Textual analysis in accounting and finance: A survey. Journal of Accounting Research, 54(4), 1187-1230.

Harry
